src/hexo/themes/documentation/source/images/results-perf.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 72 72" style="enable-background:new 0 0 72 72;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#2D28A1;}
+	.st1{fill:#817EC7;}
+	.st2{fill:#5753B4;}
+	.st3{fill:#0B0687;}
+	.st4{fill:#ACADDA;}
+	.st5{fill:none;stroke:#4046DD;stroke-width:4;stroke-linecap:round;stroke-linejoin:round;}
+</style>
+<title>icons</title>
+<g>
+	<path class="st0" d="M38.4,33.6c-3.5-2.9-7.8-4.6-12.3-4.9v7.8c2.4,0.2,4.8,1.1,6.7,2.7L38.4,33.6z"/>
+	<path class="st1" d="M10,35.1c-3.3,3.4-5.3,7.8-5.7,12.5h8.3c0.3-2.5,1.3-5,3-6.9L10,35.1z"/>
+	<path class="st2" d="M23.9,28.7c-4.5,0.2-8.8,1.9-12.3,4.9l5.6,5.6c1.9-1.5,4.2-2.5,6.7-2.7L23.9,28.7z"/>
+	<path class="st3" d="M40,35.1l-5.6,5.6c1.7,1.9,2.7,4.3,3,6.9h8.3C45.2,42.9,43.2,38.5,40,35.1z"/>
+	<path class="st4" d="M7,32c-4.1,4.2-6.6,9.7-7,15.6h4.3c0.4-4.7,2.4-9.1,5.7-12.5L7,32z M23.9,24.4c-5.7,0.2-11.1,2.4-15.4,6.1
+		l3.1,3.1c3.5-2.9,7.8-4.6,12.3-4.9L23.9,24.4z M41.4,30.5c-4.3-3.7-9.7-5.9-15.3-6.1v4.3c4.5,0.2,8.8,1.9,12.3,4.9L41.4,30.5z
+		 M50,47.6c-0.4-5.9-2.9-11.4-7-15.6l-3,3c3.3,3.4,5.3,7.8,5.7,12.5L50,47.6z"/>
+	<path class="st5" d="M39.7,42.3l-14.2,4.8"/>
+</g>
+</svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72 72"><style>.a{fill:#2d28a1}.b{fill:#817ec7}.c{fill:#5753b4}.d{fill:#0b0687}.e{fill:#acadda}.f{fill:none;stroke:#4046dd;stroke-width:4;stroke-linecap:round;stroke-linejoin:round}</style><path d="M38.4 33.6c-3.5-2.9-7.8-4.6-12.3-4.9v7.8c2.4.2 4.8 1.1 6.7 2.7l5.6-5.6z" class="a"/><path d="M10 35.1c-3.3 3.4-5.3 7.8-5.7 12.5h8.3c.3-2.5 1.3-5 3-6.9L10 35.1z" class="b"/><path d="M23.9 28.7c-4.5.2-8.8 1.9-12.3 4.9l5.6 5.6c1.9-1.5 4.2-2.5 6.7-2.7v-7.8z" class="c"/><path d="M40 35.1l-5.6 5.6c1.7 1.9 2.7 4.3 3 6.9h8.3c-.5-4.7-2.5-9.1-5.7-12.5z" class="d"/><path d="M7 32C2.9 36.2.4 41.7 0 47.6h4.3c.4-4.7 2.4-9.1 5.7-12.5L7 32zm16.9-7.6c-5.7.2-11.1 2.4-15.4 6.1l3.1 3.1c3.5-2.9 7.8-4.6 12.3-4.9v-4.3zm17.5 6.1c-4.3-3.7-9.7-5.9-15.3-6.1v4.3c4.5.2 8.8 1.9 12.3 4.9l3-3.1zM50 47.6c-.4-5.9-2.9-11.4-7-15.6l-3 3c3.3 3.4 5.3 7.8 5.7 12.5l4.3.1z" class="e"/><path d="M39.7 42.3l-14.2 4.8" class="f"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?